docs/annexes/annex-06-pid-rulebook.md

```diff
@@ -491,16 +517,116 @@ If other CDDL representation types will be used in future versions of
+| `birth_city` | `place_of_birth/locality` | |
+| `resident_address` | `address/formatted` | |
+| `resident_country` | `address/country` | |
+| `resident_state` | `address/region` | |
+| `resident_city` | `address/locality` | |
+| `resident_postal_code` | `address/postal_code` | |
+| `resident_street` | `address/street_address` | *) |
+| `resident_house_number` | `address/house_number` | *) |
+| `gender` | `gender` | Data type mismatch - JWT claims use `male`, `female` and custom text values. *) |
+| `nationality` | `nationalities` | Defined as an array, here containing only one entry. |
+| `issuance_date` | `iat` | |
+| `expiry_date` | `exp` | |
+
+*Table 3: Data Element Identifier to Claim Name Mapping*
+
+Fields marked with (*) indicate that there is not a one-to-one mapping
+between the data element and a suitable pre-defined JWT claim.
+This should be resolved in future versions of this document.
+
+EXAMPLE: The following example shows a PID in SD-JWT VC format.
+
+```json
+{
+    "vct": "https://memberstate.example/credential/pid",
+    "vct#integrity": "sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM",
+
+    "given_name": "Jean",
+    "family_name": "Dupont",
+    "birthdate": "1980-05-23",
+
+    "age_equal_or_over": {
+        "12": true,
+        "14": true,
+        "16": true,
+        "18": true,
+        "21": true,
+        "65": false
+    },
+    "age_in_years": 44,
+    "age_birth_year": 1980,
+
+    "address": {
+        "street_address": "123 Via Appia",
+        "locality": "Rome",
+        "region": "Lazio",
+        "postal_code": "00100",
+        "country": "IT"
+    },
+
+    "nationalities": ["FR"],
+
+    "gender": "male",
+
+    "place_of_birth": {
+        "locality": "Leipzig",
+        "region": "Saxony",
+        "country": "DD"
+    },
+
+    "cnf": {
+        "jwk": {
+            "kty": "EC",
+            "crv": "P-256",
+            "x": "52aDI_ur05n1f_p3jiYGUU82oKZr3m4LsAErM536crQ",
+            "y": "ckhZ-KQ5aXNL91R8Eufg1aOf8Z5pZJnIvuCzNGfdnzo"
+        }
+    }
+}
+```
+
+Note: The `cnf` claim is used for expressing key binding in SD-JWT VCs.
+The example above shows a public key in JWK format.
+
+Note: Additional technical claims are not shown here, including
+references to the issuer, validity status information, and more.
+
+
+##### 2.6.3.3 Supplementary Details
+
+SD-JWT VCs containing data elements in Table 1 and Table 2 SHALL be
+released only in a VP Token, as specified in \[OpenID4VP\]. This means
+that these data elements SHALL be signed by the PID Provider, not by the
+Wallet Instance.
 
 At the discretion of the PID Provider, domestic data elements (see
 section 2.2.2) MAY be released in either a VP Token or an ID Token.
 
+PID Issuers SHALL make all data elements in Table 1 and Table 2 except
+for `issuance_date`, `expiry_date` and `issuing_country` available for
+selective disclosure, including parent and child claims where
```

Comment:
I was wondering whether rulebook should be explicit not only if elements are always or selectively disclosable, but in addition in the SD case, which sd-jwt option (flat, structured, recursive) should be used for every element (especial objects and arrays)

Reply:
This is covered by the sentence specifying that parent and child elements shall be selectively disclosable, which implies the "recursive" option. (Note, that SD-JWT doesn't specify the three options you listed, it just uses them as examples. Therefore I didn't use the word "recursive" here.)

Reply:
> This is covered by the sentence specifying that parent and child elements shall be selectively disclosable, which implies the "recursive" option. 

The "parent and child elements shall be selectively disclosable" is also satisfied with "structured" option, IMO, for objects that have only a single level of children (like `address`). So I think it would be better to explicitly specified.
 
Note: Indeed I have the feeling that SD-JWT lacks some terminology as to how you calculate disclosures (for both objects, arrays) with these 3 options. Examples are not enough, because as in this case somebody cannot formally refer to them.

Reply:
I added a change proposal to the text to make the wording more clear. Please let me know what you think!

The lack of a naming for the options in SD-JWT is very much intentional. The options are just meant as examples, not to say that a credential has to follow one principle or the other. Instead, for every element individually, the issuer has to decide whether to make it selectively disclosable or not.